integration-libs/cdc/components/login/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './facade/cdc-login-form.module';
+export * from './facade/index';
```

Comment:
The files CdcLoginFormModule and CdcLoginFormComponent are not facades. I guess it's a leftover name after a few refactors.

Let's rename the folder here from `/facade` for `/login-form`

Reply:
okay Kris